src/main/java/com/codingiscaring/Inventory.java

```diff
@@ -0,0 +1,37 @@
+package com.codingiscaring;
+
+import java.util.List;
+
+public class Inventory {
+
+    private List<Item> items;
+
+    public Inventory(List<Item> items) {
+        this.items = items;
+    }
+
+    private Item findItemBy(String code) {
+        return items.stream()
+                .filter(i -> i.getCode().equals(code))
+                .findFirst()
+                .orElse(null);
+    }
+
+    private void updateInventory(Item item) {
+        item.decreaseQuantity();
+    }
+
+    public Item sell(String code, double money) throws InventoryException {
+        Item item = findItemBy(code);
+
+        if (item == null)
+            throw new InventoryException("Invalid selection! : Money in vending machine = " + DoubleUtils.twoDecimalsFormat(money));
```

Comment:
```suggestion
            throw new InvalidSelectionException("Invalid selection!");
```

Create new class InvalidSelectionException